roles/elastic_search/tasks/main.yaml

```diff
@@ -9,3 +9,5 @@
 #- include: 130_docker_python_libs.yaml
 
 - include: 150_elastic_search_docker_container.yaml
+
+# - include: 160_elastic_search_postconfig.yaml
```

Comment:
We decided to follow the second way, i.e. services are responsible for creating indexes which they use.